test/test_optim.py

```diff
@@ -320,6 +320,34 @@ def _test_complex_optimizer(self, optimizer_constructor):
 
             self.assertEqual(torch.view_as_real(complex_param), real_param)
 
+    def _test_complex_2d(self, optimizer_constructor, f=None):
```

Comment:
The `f` argument is not actually needed right?

Reply:
I want to reuse this function for all tests involving the complex optimisers.